apps/opik-backend/src/main/java/com/comet/opik/domain/llmproviders/LlmProviderStreamHandler.java

```diff
@@ -0,0 +1,9 @@
+package com.comet.opik.domain.llmproviders;
+
+import org.glassfish.jersey.server.ChunkedOutput;
+
+public interface LlmProviderStreamHandler {
```

Comment:
Seems unnecessary, probably no need to multiple stream handlers.

Reply:
Done in https://github.com/comet-ml/opik/pull/957/commits/8b7025b3dbfa65243f742c192b90c6a0666aa99b